lib/param_parsing/parameters_test.go

```diff
@@ -10,6 +10,19 @@ import (
 	"github.com/warrensbox/terraform-switcher/lib"
 )
 
+func TestGetParameters_arch_from_args(t *testing.T) {
+	expected := "amd64"
```

Comment:
```suggestion
	expected := "amd64_from_arg"
```
Could this be an arbitrary value to avoid it matching a default value from a real system?

Reply:
Wait how does this test even work? The argument `amd64` is being supplied directly? Won't this be a positional argument and be used for the `version` argument and so the test is just validating that default argument?

Reply:
That's another copy-pasta test from the code right below it. With arbitrary value it was failing for me, so I set it to a correct value.
I'm a way far from being familiar with this testing stuff unfortunately. Would you be up to help reviewing and streamlining these tests please? (or I can remove this specific one and we can add it back another time)

Reply:
Yep, but the one below works because it's testing the position arg for version. I think you just need:
```
os.Args = []string{"cmd", "--arch=" + expected}
```

Reply:
Ah, I see now. Thanks for the pointer 👍🏻 Addressed via a12fc57